SU2_CFD/include/solvers/CSolver.hpp

```diff
@@ -4205,8 +4205,16 @@ class CSolver {
    * \param[in] config - Definition of the particular problem.
    * \param[in] referenceCoord - Determine if the mesh is deformed from the reference or from the current coordinates.
    */
-  inline virtual void SetMesh_Stiffness(CGeometry **geometry,
-                                        CNumerics **numerics,
+  inline virtual void DeformMesh(CGeometry *geometry,
```

Comment:
The main issue in integrating the CMeshSolver in CDeformationDriver was that several routines in the solver implementation are specifically written to handle multigrid. To circumvent these function calls, it seemed simpler as a first approach to overload the DeformMesh function so that, if a simple pointer to a geometry is given (rather than a double pointer), a multigrid-free implementation is used.